app/Policies/HoquJobPolicy.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Policies;
+
+use App\Models\HoquJob;
+use App\Models\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class HoquJobPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view any models.
+     *
+     * @param  \App\Models\User  $user
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function viewAny(User $user)
+    {
+        return true;
+    }
+
+    /**
+     * Determine whether the user can view the model.
+     *
+     * @param  \App\Models\User  $user
+     * @param  \App\Models\HoquJob  $hoquJob
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function view(User $user, HoquJob $hoquJob)
+    {
+        return true;
+    }
+
+    /**
+     * Determine whether the user can create models.
+     *
+     * @param  \App\Models\User  $user
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function create(User $user)
+    {
+        return $user->email === "team@webmapp.it";
```

Comment:
Meglio mettere tutto nel before ome già fatto in altri casi per le piattaforme con unico utente di accesso